apps/condo/domains/meter/components/CreateMeterReadingsForm.tsx

```diff
@@ -57,12 +57,11 @@ type MetersTableRecord = {
     tariffNumber: string
 }
 
-function getTableData (meters: MeterType[] | PropertyMeterType[], meterReadings: MeterReadingType[] | PropertyMeterReadingType[]): MetersTableRecord[] {
+function getTableData (meters: MeterType[] | PropertyMeterType[], meterReadings): MetersTableRecord[] {
     const dataSource: MetersTableRecord[] = []
 
-    // @ts-ignore
-    const lastMeterReadings = uniqWith(meterReadings,
-        (meterReading1, meterReading2) =>
+    const lastMeterReadings = uniqWith(meterReadings.filter((meterReading) => meterReading.meter !== null ),
```

Comment:
`meterReadings` can be without `meter`?
`meter` is a required field

Reply:
yes, when the user deletes `meter`, a worker task is launched in the `afterChange` that deletes all its `MeterReadings`. Therefore, there is some delay when `readings` are still available, but the field `meter` already has a deleted `meter` and on the front it is not available on the front